test/ReadsUtils_server_test.py

```diff
@@ -561,8 +581,18 @@ def md5(self, filename):
          'single_genome': None
          })
 
-    # FASTA/Q tests ########################################################
+    @classmethod
+    def start_local_ftp_connection(cls, fq_filename):
```

Comment:
A more accurate name would be `upload_file_to_local_ftp_server` or something like that, but *shrug*